devolutions-gateway/src/api/network_scan.rs

```diff
@@ -105,15 +105,14 @@ impl From<NetworkScanQueryParams> for NetworkScannerParams {
 pub struct NetworkScanResponse {
     pub ip: IpAddr,
     pub hostname: Option<String>,
-    #[serde(rename = "type")]
-    pub ty: ApplicationProtocol,
+    pub ap: ApplicationProtocol,
```

Comment:
Copying my comment from Slack for visibility:

I was really suggesting to use `protocol` or `application_protocol`, not `ap` which is used as an abbreviation in the token claim jet_ap  (because JWT claims are abbreviated by convention).
Here, it’s okay to not use the abbreviation since it’s not a token claim.

I think that `protocol` is a good compromise if you prefer to avoid the longer `application_protocol` naming.

Reply:
use `protocol` instead, `application_protocol` is a bit to verbose